autoload/easy_align.vim

```diff
@@ -89,9 +89,33 @@ function! s:floor2(v)
   return a:v % 2 == 0 ? a:v : a:v - 1
 endfunction
 
+function! s:get_highlight_group_name(line, col)
+  let l:hl = synIDattr(synID(a:line, a:col, 0), 'name')
+
+  if l:hl == '' && has('nvim')
+    " try getting the highlighting group using neovim treestiiter approach.
+	" The required functionality appeared in neovim nightly around 18 Dec, 2022
+	" https://www.reddit.com/r/neovim/comments/zou5ig/inspect_is_now_available_on_nightly/
```

Comment:
We can use `exists(':Inspect')` in the condition if this functionality came along with the command. Or, preferably, if you know the exact version number which is required, `has('nvim-0.x.y')`.

Reply:
I will be adding the exact neovim version, as asked. With that being said, the check similar to `exists(':Inspect')` is already happening in lua: `'vim.inspect_pos and ...` means "do something if the function vim.inspect_pos exists".

The comment is advisory as to why this method may not work ...

Reply:
Updated, please review